arelle/ValidateDuplicateFacts.py

```diff
@@ -210,6 +210,8 @@ def deduplicateConsistentPairs(self) -> list[ModelFact]:
         if not self.areNumeric:
             # Consistency is equivalent to completeness for non-numeric facts
             return facts
+        if len(facts) < 2:
+            return facts
```

Comment:
Out of curiosity, what issue is this fixing?  Is it just skipping a bit of working building the map?

Reply:
This came up because I encountered a scenario where a complete duplicate subset consisted of all nil facts. This results in `deduplicateCompleteSubsets` returning a single fact. The `decimals is not None` assertion then fails, but we really don't even need to be iterating over the facts at all when there's a single fact.